src/shared/install.c

```diff
@@ -1995,8 +1995,6 @@ static int install_info_symlink_wants(
                                 return install_changes_add(changes, n_changes, -EIDRM, dst, n);
                         else
                                 return install_changes_add(changes, n_changes, -EUCLEAN, dst, NULL);
-
-                        continue;
```

Comment:
No, this is wrong. `return` was added inadvertently in 94e7298d309fef7710174def820e9d38e512a086. We want to do the full loop here and collect any errors in `changes`.